g2p_programs/models/payment.py

```diff
@@ -178,7 +178,3 @@ class G2PPaymentBatchTag(models.Model):
     domain = fields.Text(default="[]")
 
     max_batch_size = fields.Integer(default=500)
-
```

Comment:
how is the constraint added?

Reply:
@shivamg9  fixed the ERN issue in odoo15 so based on that i fixed it here